vivarium/core/store.py

```diff
@@ -1064,13 +1073,14 @@ def _delete_path(self, path):
         if not path:
             self.inner = {}
             self.value = None
-            return self
+            return None
```

Comment:
is this change required because `self` is deleted?

Reply:
Nah I just thought it wasn't really necessary. We don't make use of the return value anywhere and this is a private function.

Reply:
Can you bring it back? It might be part of the store api developed with @WilliamIX. Then we're good to merge